evm_loader/program/src/debug.rs

```diff
@@ -15,3 +15,34 @@ macro_rules! debug_print {
 macro_rules! debug_print {
     ($( $args:expr ),*) => {};
 }
+
+#[cfg(target_os = "solana")]
+macro_rules! log_msg {
```

Comment:
What is the difference between `debug_print` and `log_msg`?

Reply:
`debug_print` doesn't show any line in program, `log_msg` is replacement for `solana_program::msg` macros.